pyproject.toml

```diff
@@ -214,17 +247,45 @@ supported_nautobot_versions = [
     "2.0.0"
 ]
 
-[tool.pydocstyle]
-convention = "google"
-inherit = false
-match = "(?!__init__).*\\.py"
-match-dir = "(?!tests|migrations|development)[^\\.].*"
-# D212 is enabled by default in google convention, and complains if we have a docstring like:
-# """
-# My docstring is on the line after the opening quotes instead of on the same line as them.
-# """
-# We've discussed and concluded that we consider this to be a valid style choice.
-add_ignore = "D212"
+[tool.ruff]
+line-length = 120
+target-version = "py38"
+
+exclude = [
+    ".eggs",         # exclude a few common directories in the
+    ".git",          # root of the project
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "nautobot_ssot/integrations/servicenow/third_party"
+]
+
+[tool.ruff.lint]
+select = [
+    "DJ",  # flake8-django
```

Comment:
If you're replacing `pydocstyle` then you should enable ruff's `D` rules here as well!

Reply:
Good catch, I misread `pycodestyle` on the others and mistakenly thought it was already enabled. Have enabled it now and moved settings from ini into pyproject.toml. For some reason enabling it triggered a bunch of warnings. Not really sure why they didn't come up with our previous implementation of pydocstyle.